src/Stats.cpp

```diff
@@ -0,0 +1,131 @@
+#include "Stats.h"
+#include <limits>
+
+namespace mms{
+
+Stats::Stats() {
+    penalty = 0;
+}
```

Comment:
You should initialize all member variables in the initialization list, like this:
```cpp
Stats::Stats() : startedRun(false), solved(false), penalty(0.0) {
}
```